client/src/components/Home/Home.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Home.css';
-
+import shadow from './blue_blur.png';
 function Home() {
   return (
     <div className="home-component">
-      <h1>This is the Home page.</h1>
+      <img className="lander-img img-fluid" src={shadow} alt="lander"/>
```

Comment:
Using `react-lazyload` would help (I guess) as suggested by @ankiiitraj

Reply:
Will make use of it from next time, it was only for visualizing the navbar with a background other than white :-)